src/main/java/com/google/api/codegen/discogapic/MainDiscoGapicProviderFactory.java

```diff
@@ -49,16 +50,25 @@
                 .setPrefix("src/main/java")
                 .setShouldAppendPackage(true)
                 .build();
-        DiscoGapicProvider mainProvider =
+        DiscoGapicProvider clientProvider =
             DiscoGapicProvider.newBuilder()
                 .setDocument(document)
                 .setProductConfig(productConfig)
                 .setSnippetSetRunner(new CommonSnippetSetRunner(new JavaRenderingUtil()))
                 .setDocumentToViewTransformer(
                     new JavaDiscoGapicSurfaceTransformer(javaPathMapper, packageConfig))
                 .build();
+        DiscoGapicProvider messageProvider =
+            DiscoGapicProvider.newBuilder()
+                .setDocument(document)
+                .setProductConfig(productConfig)
+                .setSnippetSetRunner(new CommonSnippetSetRunner(new JavaRenderingUtil()))
+                .setDocumentToViewTransformer(
+                    new JavaDiscoGapicSchemaToViewTransformer(javaPathMapper, packageConfig))
```

Comment:
JavaDiscoGapicSchemaToViewTransformer creates the ViewModel for Discovery doc schemas, e.g. a compute Disk; each schema file just has getters and setters, nothing fancy.
JavaDiscoGapicSurfaceTransformer creates the ViewModel for the API client file, which is what the user interacts with to make the REST calls; it could have methods like ListDisks().